sale_order_secondary_unit/models/sale_order.py

```diff
@@ -11,52 +12,44 @@ class SaleOrderLine(models.Model):
         "uom_field": "product_uom",
     }
 
-    secondary_uom_unit_price = fields.Float(
+    secondary_uom_price_unit = fields.Float(
```

Comment:
Please add a migration script renaming this field, or let it as it was.

Reply:
What's the point if it was computed and not stored? And now is computed and stored?

Reply:
Was it not stored? And why is not stored now? I don't think the store attribute should change then.

Reply:
It was not stored, it is stored now. Truth be told - the field is not used anywhere since removal of that one from the UI - we've talked about that not that long ago.

Reply:
Then don't do it stored if it's even not used in UI.

Reply:
OK, I didn't know exactly the history, but I think this is not needed at all.

Reply:
Well, I guess since it's not in the UI since earlier version, I'll just drop the field entirely

Reply:
Or expose it in the UI..

Reply:
Yeah, I'll expose it in the UI. I don't see why migration would be needed, if that's a computed-stored field. Also, it's stored and precomputed alike price_unit